cupyx/jit/_builtin_funcs.py

```diff
@@ -50,6 +51,50 @@ def call(self, env, *args, **kwargs):
         return Range(start, stop, step, ctype, step_is_positive)
 
 
+class Len(BuiltinFunc):
+    def call(self, env, *args, **kwds):
+        if len(args) > 1:
+            raise TypeError(f'len expects only 1 argument, got {len(args)}')
+        arg = args[0]
+        if isinstance(arg.ctype, _cuda_types.CArray):
+            return Data(f'{arg.code}.size()', _cuda_types.PtrDiff())
+        if isinstance(arg.ctype, _cuda_types.SharedMem):
+            if arg.ctype._size is None:
+                raise TypeError('unsized shared memory is not supported')
+            return Constant(arg.ctype._size)
+        raise TypeError('len supports only array type')
+
+
+class Min(BuiltinFunc):
+    def call(self, env, *args, **kwds):
```

Comment:
How about calling `_compile._call_ufunc(cupy.minimum, args, None, env)` for simplification?

Reply:
I suppose `cupy.maximum` only support two arguments... or you mean calling it for multiple times?

Reply:
Yes! I mean calling `cupy.maximum` n-1 times.

Reply:
So we prefer ufunc to math api?

Reply:
Yes, I prefer ufunc because Python's built-in min/max behavior is more similar to ufunc than CUDA math API.
In the future, we will create a module for CUDA math API and intrinsic functions.